src/crypto-providers/trezorCryptoProvider.ts

```diff
@@ -109,6 +108,18 @@ export const TrezorCryptoProvider: () => Promise<CryptoProvider> = async () => {
           throw Error(Errors.TrezorPassphraseNotInsertableOnDevice)
         }
       }
+
+      // Connect expects an explicit confirmation that no backup was created
+      // for cardanoGetPublicKey call
+      if (
+        event.type === TrezorTypes.UI.REQUEST_CONFIRMATION &&
+        event.payload.view === 'no-backup'
+      ) {
+        TrezorConnect.uiResponse({
+          type: TrezorTypes.UI.RECEIVE_CONFIRMATION,
+          payload: true,
+        })
+      }
```

Comment:
Should we at least print a warning? @janmazak what do you think?

Reply:
mhh... the warning is already on the hw gui. it would interrupt the currently used workflow if there is an additional warning output returned by cardano-hw-cli.

Reply:
I don't understand what "backup" refers to. But if a warning is shown on the device, I think that's enough.

Reply:
> if there is no backup, the hw-gui shows a prominent line at the top of the screen ⚠️BACKUP NEEDED all the time.

True, I just wonder whether we can somehow accomodate this new expectation from Connect (the user explicitly confirming this in the client UI)...

I know that doing a `y/n` dialog in hw-cli is nonsense (for example in the context of your scripts). Would also just printing a warning message (without interaction) complicate your flow?

Reply:
> I don't understand what "backup" refers to.

Writing down the mnemonic genereated by Trezor. (You can choose to skip this when setting up the device, but then the yellow warning is displayed when it's idle.)

Reply:
It would yes, could you - if really needed - return that information together with the device version like:
```
cardano-hw-cli device version
Trezor app version 2.6.5 BACKUP NEEDED
```

But here are two pics of the hw gui, i don't think you don't have to show that extra warning within hw-cli at all.

![image](https://github.com/vacuumlabs/cardano-hw-cli/assets/47434720/8804181d-19b0-4eee-88ad-587e72da5d3d)
![image](https://github.com/vacuumlabs/cardano-hw-cli/assets/47434720/28e974c3-46bf-4e22-9d0d-a800db029951)

Reply:
Okay, so let's just stick with the warning being displayed on screen :+1: